src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+    private static final String DELIMITERS = "\\s*[^a-zA-Z]+\\s*";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder information = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String value = bufferedReader.readLine();
+            if (value == null) {
+                return new String[]{};
+            }
```

Comment:
it's duplicated check , let's leave only while condition

Reply:
done